src/server_manager/install_scripts/install_server.sh

```diff
@@ -235,11 +268,17 @@ install_shadowbox() {
   # require new dependencies.
   cat <<END_OF_SERVER_OUTPUT
 
-Please copy the following configuration to your Outline Manager:
+CONGRATULATIONS! Your Outline Server is up and running.
+
+${FIREWALL_STATUS}
+To manage your Outline Server, please copy the following text (including curly
+brackets) into Step 2 of the Outline Manager interface:
```

Comment:
Presumably they've just come from the Outline Manager; I would re-word to something like: "into the Outline Manager".

Reply:
The same screen on the Manager shows both Step 1 and Step 2. I think mentioning Step 2 may feel excessive, but makes it more explicit. The text was suggested by Santi.

Reply:
I'm not going to hold this up on worsmithing but FYI "the Outline Manager interface" is redundant - it's just "the Outline Manager".